pkg/controller/priorityqueue/priorityqueue_test.go

```diff
@@ -283,6 +284,41 @@ var _ = Describe("Controllerworkqueue", func() {
 		Expect(metrics.depth["test"]).To(Equal(0))
 		Expect(metrics.adds["test"]).To(Equal(2))
 	})
+
+	It("returns many items", func() {
```

Comment:
Not sure how to verify that the spin goroutine didn't panic.

Locally with Intellij, I've hit a panic break point and after continuing the test was shown as successful. I'm not sure if the same happens in CI

Reply:
Maybe that there was a breakpoint caused go test to not recognize it properly?

Reply:
Jup maybe